client/src/components/overview/components/newButton.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+let NewButton = (props) => {
+  if (props.availableAmount === '0') {
+    return (
+      <div></div>
```

Comment:
I love how you are handling possible missing data with these `<div></div>s`. Something I need to do!